config/locales/es.yml

```diff
@@ -178,6 +178,11 @@ es:
     file_type: Por favor, sube un tipo de documento válido. Los tipos aceptados incluyen jpg, pdf y png.
     zip: Por favor, introduzca un código postal válido de 5 dígitos.
   views:
+    clients:
```

Comment:
IMHO let's either decide to translate all of admin, or punt on that entirely. I'm worried we'll have a half-translated admin site that looks weird.

Reply:
DECIDED with @bengolder & @jenny-heath that we will i18n-ify all new strings in admin, and we're willing to leave English-language text in `es.yml` to get there fast, and Ben Golder might possibly translate some to Spanish but it's not something we need others to necessarily do.